### ISSUE TITLE
加入文字信息

### ISSUE THREAD

**Issue by milk333445** [ppaanngggg/layoutreader#12]
您好，想請問如果想要加上文字信息進行訓練的話，是要在構造輸入數據集時，也就是構造input_ids的時候，用layoutlm3的tokenizer轉成token放入嗎，除此之外還有其餘需要注意修改的點嗎，謝謝!

**Comment by ppaanngggg** [comment]
可以试试，一个span可能会被tokenizer分成多个token，这样对应的位置的embedding和训练目标也都需要重复多遍

**Comment by ppaanngggg** [comment]
还有什么问题吗? 没有的话我关闭issue了. 另外你这个需求可以问问他 https://github.com/ppaanngggg/layoutreader/issues/8

**Comment by milk333445** [comment]
好的大概沒問題了，謝謝您的回覆！謝謝您

**Comment by cloudfool** [comment]
> 可以试试，一个span可能会被tokenizer分成多个token，这样对应的位置的embedding和训练目标也都需要重复多遍

可以再解释一下吗？一个span被tokenizer分成多个token，那相应的bbox,labels,attention_mask也要分成多个？
